### PR TITLE
New dev

### PR DESCRIPTION
**Pull request by JohanBester**
Merge New-dev back into development

### CODE REVIEW

File: AddendumOfResources/AddendumOfResources.md [hunk 1]
```diff
@@ -328,10 +328,6 @@ Create a profile for yourself and select your areas of interest
 
 ## Python Resources
 
-<<<<<<< HEAD
```
Comment (matsinet) [inline AddendumOfResources/AddendumOfResources.md:331]: Are code merge markers being removed?

File: AddendumOfResources/AddendumOfResources.md [hunk 2]
```diff
@@ -367,8 +363,6 @@ Best resources for learning Python for beginners
 - [Scientific Computing with Python (freeCodeCamp)](https://www.freecodecamp.org/learn/scientific-computing-with-python/)
     In this freeCodeCamp certification course, you will learn about loops, lists, dictionaries, networking, web services and more. You will also have the opportunity to build five projects: Arithmetic Formatter, Time Calculator, Budget App, Polygon Area Calculator, and Probability Calculator.
 
->>>>>>> main
```
Comment (matsinet) [inline AddendumOfResources/AddendumOfResources.md:370]: Are code merge markers being removed?

**Review by matsinet — APPROVED** [review]

File: AddendumOfResources/AddendumOfResources.md (hunk 1 above)
Comment (JohanBester) [reply AddendumOfResources/AddendumOfResources.md:331]: Yes they are being removed

File: AddendumOfResources/AddendumOfResources.md (hunk 2 above)
Comment (JohanBester) [reply AddendumOfResources/AddendumOfResources.md:370]: Yes these marks are being removed

**Pull request merged**